### ISSUE TITLE
Give some hints on what's wrong when tasks fail to install due to `ImportError`

### ISSUE THREAD

**Issue by dialelo** [jgorset/django-kronos#9]
When registering a cronjob, if any import is wrong and raises an `ImportError`, the job won't be registered and there is no indication of what failed after running `installtasks`:

```py
import kronos

raise ImportError

kronos.register('@hourly'):
def i_wont_install():
    print "I won't install"
```

```sh
$ python manage.py installtasks
Installed 0 tasks
```

It would be nice to have an error message indicating why registering the task failed.

Thank you for `django-kronos`!

**Comment by joshblum** [comment]
Closed by #24

**Comment by jgorset** [comment]
Indeed. Thank you.
